### ISSUE TITLE
A question about [GO[ token

### ISSUE THREAD

**Issue by zhaiyukun** [roatienza/deep-text-recognition-benchmark#19]
criterion = torch.nn.CrossEntropyLoss(ignore_index=0).to(device)  # ignore [GO] token = ignore index 0

why you ignore GO token when setup loss?

Thank you

**Comment by zhaiyukun** [comment]
thank you for you reply!
